.github/workflows/cypress.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 concurrency: 
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
I think this reverts a change @qgerome  made to give us some performance gain

Reply:
This change in particular was not for performance but to cancel already running workflows when you trigger a new one. But there is an error in the `group` and it cancels all runs of this workflow instead. We can safely merge this.